overlord/servicestate/quota_handlers.go

```diff
@@ -889,12 +889,12 @@ func affectedQuotasForQuotaAddSnap(t *state.Task) (quotas []string, err error) {
 }
 
 func affectedSnapsForQuotaAddSnap(t *state.Task) (snaps []string, err error) {
```

Comment:
you need to double check, but we might not need this code anymore, snapstate/conflict.go has defaul support for SnapSetup-using tasks

Reply:
Yea you are absolutely right, I removed this.